### PR TITLE
Add Stars badges for GitHub repositories

### PR DESCRIPTION
**Pull request by ErezBiren**
Add ranking stars badges (Only for GitHub repositories)

solve #113

### CODE REVIEW

**Comment by UrielOfir** [comment]
Hi Erez, thank you for your contribution :)
Please fix the conflict. In your PR you don't have the main language badge, please update it from the current main branch.

**Comment by UrielOfir** [comment]
resolves #113

File: README.md [hunk 1]
```diff
@@ -24,106 +24,190 @@ A curated list of awesome Open Source Software made in Israel, and by Israelis.
 
 ## Projects
 * [@lirantal/typeform-client](https://github.com/lirantal/typeform-client) - a friendlier Typeform Node.js API client ![GitHub last commit](https://img.shields.io/github/last-commit/lirantal/typeform-client?style=flat-square)
+![GitHub stars](https://img.shields.io/github/stars/lirantal/typeform-client?style=flat-square)
```
Comment (UrielOfir) [inline README.md:27]: main language badge is missing.

**Review by UrielOfir — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ErezBiren) [reply README.md:27]: Thanks! 
Added

**Comment by ErezBiren** [comment]
> Hi Erez, thank you for your contribution :) Please fix the conflict. In your PR you don't have the main language badge, please update it from the current main branch.

Thanks! hope I fixed it right this time

**Review by UrielOfir — APPROVED** [review]
Thank you for your contribution :)

**Pull request merged**